.github/ISSUE_TEMPLATE/issue-template.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Issue Template
```

Comment:
We could rename this to `Bug Reports`, if it's clearer to the user. eg: 
![image](https://user-images.githubusercontent.com/41563608/99234296-5d080700-281a-11eb-9a8a-1c09af710b5a.png)

Reply:
Updating to Bug Reports sounds good to me